app/routes.py

```diff
@@ -1,2 +1,184 @@
-from flask import Blueprint
+from werkzeug.wrappers import ETagRequestMixin
+from app.models.goal import Goal
+from flask import Blueprint, request, make_response, jsonify
+from app import db
+from app.models.task import Task
+from sqlalchemy import asc, desc
+from datetime import datetime
+import requests
+import os
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["GET","POST"])
+def handle_tasks():
+    if request.method == "GET":
+        sort = request.args.get("sort")
+
+        if sort == "asc":
+            tasks = Task.query.order_by(asc("title"))
+        
+        elif sort == "desc":
+            tasks = Task.query.order_by(desc("title"))
+        
+        else:
+            tasks = Task.query.all()
+
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append({
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": task.is_complete()
+            })
+        return jsonify(tasks_response)
+
+    elif request.method == "POST":
```

Comment:
With regards to the point in separating out the functions for each route (that you noted in Learn), this way of organizing your code can increase readability by reducing indentation and making meaningful function names. For instance, if we separate out GET and POST, we might call one function `def all_tasks()` and the other `def create_task()`.